test/misc.js

```diff
@@ -26,10 +27,9 @@ describe('Misc', () => {
     const stream = util.file('small')
 
     // Don't let FormData figure out a filename
-    delete stream.fd
-    delete stream.path
+    const hidden = pipeline(stream, new PassThrough(), () => {})
```

Comment:
```suggestion
    const hidden = await promisify(pipeline)(stream, new PassThrough())
```

Reply:
that breaks contract because result of promisified pipeline will be `undefined`
https://nodejs.org/api/stream.html#streampipelinesource-transforms-destination-callback
I actually borrowed that changes from https://github.com/expressjs/multer/blob/explore-new-api/test/misc.js#L31
replaced with `stream.pipe(new PassThrough())`